### DISCUSSION TITLE
Finetuning fishspeech tts on hindi language data

### DISCUSSION THREAD

**Discussion by abhisirka2001** [fishaudio/fish-speech#856] [Show and tell]
Would love to discuss further on how to improve my results 
Here are the details about my recent finetuning trial 
9k steps
1100 hrs of data
lora rank 32
learning rate 2e-05
model started speaking hindi, however the accent is different but the results motivated me to go ahead with finetuning.

text:"जब भी हम जीवन में किसी बड़े लक्ष्य को प्राप्त करने के लिए प्रयास करते हैं, तो यह न केवल हमारी मेहनत और संकल्प को दर्शाता है, बल्कि यह भी दिखाता है कि हमने अपने अंदर की असीम क्षमताओं को पहचानते हुए हर मुश्किल और चुनौती का सामना करते हुए, चाहे वो कितनी ही कठिन क्यों न हो, आगे बढ़ने और अपने सपनों को साकार करने की शक्ति को जागृत किया है, क्योंकि सच्ची सफलता वही होती है जो हमें न केवल संतोष देती है बल्कि हमारे आसपास के लोगों को भी प्रेरित करती है।"
[https://drive.google.com/drive/folders/1bmQswEO3ES1_r4LCU8RB3xyfVh9sbj09?usp=sharing](url)

Would love to collaborate with anybody interested in indic finetuning

**Comment by viju-sharma** [comment]
> Would love to discuss further on how to improve my results Here are the details about my recent finetuning trial 9k steps 1100 hrs of data lora rank 32 learning rate 2e-05 model started speaking hindi, however the accent is different but the results motivated me to go ahead with finetuning.
> 
> text:"जब भी हम जीवन में किसी बड़े लक्ष्य को प्राप्त करने के लिए प्रयास करते हैं, तो यह न केवल हमारी मेहनत और संकल्प को दर्शाता है, बल्कि यह भी दिखाता है कि हमने अपने अंदर की असीम क्षमताओं को पहचानते हुए हर मुश्किल और चुनौती का सामना करते हुए, चाहे वो कितनी ही कठिन क्यों न हो, आगे बढ़ने और अपने सपनों को साकार करने की शक्ति को जागृत किया है, क्योंकि सच्ची सफलता वही होती है जो हमें न केवल संतोष देती है बल्कि हमारे आसपास के लोगों को भी प्रेरित करती है।" [https://drive.google.com/drive/folders/1bmQswEO3ES1_r4LCU8RB3xyfVh9sbj09?usp=sharing](url)
> 
> Would love to collaborate with anybody interested in indic finetuning

Hello,

I am interested in collaborating with you. I attempted to train a model using a 10-hour dataset and 6,000 steps, but the results were not comprehensible.

**Comment by Pandede** [comment]
I'm also finetuning the model on new language (Thai, Vietnamese), looking forward the advices from the authors.

**Comment by zyandtom** [comment]
hi guys, where is the finetuning guide? I did not see it

**Comment by rj25031** [comment]
Hey!! @abhisirka2001  i am also working on this would love to colaborate

**Comment by La-Antique** [comment]
I am currently training a LoRA but what models we should train? The official repo suggests to train on S1 mini

**Comment by La-Antique** [comment]
> Would love to discuss further on how to improve my results Here are the details about my recent finetuning trial 9k steps 1100 hrs of data lora rank 32 learning rate 2e-05 model started speaking hindi, however the accent is different but the results motivated me to go ahead with finetuning.
> 
> text:"जब भी हम जीवन में किसी बड़े लक्ष्य को प्राप्त करने के लिए प्रयास करते हैं, तो यह न केवल हमारी मेहनत और संकल्प को दर्शाता है, बल्कि यह भी दिखाता है कि हमने अपने अंदर की असीम क्षमताओं को पहचानते हुए हर मुश्किल और चुनौती का सामना करते हुए, चाहे वो कितनी ही कठिन क्यों न हो, आगे बढ़ने और अपने सपनों को साकार करने की शक्ति को जागृत किया है, क्योंकि सच्ची सफलता वही होती है जो हमें न केवल संतोष देती है बल्कि हमारे आसपास के लोगों को भी प्रेरित करती है।" [https://drive.google.com/drive/folders/1bmQswEO3ES1_r4LCU8RB3xyfVh9sbj09?usp=sharing](url)
> 
> Would love to collaborate with anybody interested in indic finetuning

would you mind sharing your training parameters?
